pkg/proxy/apis/config/v1alpha1/defaults.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2015 The Kubernetes Authors.
+Copyright 2018 The Kubernetes Authors.
```

Comment:
Moving and changing the existing files don't necessarily change the year here, IMO. @bgrant0607 WDYT?